azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/components/ChatCompsiteActionBar.kt

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.presentation.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.clipRect
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.core.graphics.toColorInt
+import com.azure.android.communication.ui.chat.R
+import com.azure.android.communication.ui.chat.presentation.components.microcomponents.ChatCompositeActionBarBackButton
+
+@Composable
+fun ChatCompsiteActionBar(
```

Comment:
We will checkin the action bar required as per figma with actual colors. We do not need to checkin temp code into develop.

Reply:
Checked in a different basic action bar component UI to demonstrate an example of adding any component.